README.md

```diff
@@ -31,6 +31,13 @@ You will find both
 [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) repository.
 - the examples described in more details below in this page. They demonstrate how to use a single feature.
 
+## Demo
```

Comment:
@tbouffard Is it ok four you ?

Reply:
👍🏿 perfect

Reply:
the title miss an emoji for consistency with others.

Reply:
the `a `demo` that shows several features of `bpmn-visualization` at the same time. The sources of the demo are available in the` sentence above should be reworked.

Several demo + remove ref to the repository

Reply:
Yes, I forgot this 🙁